src/tokenizer/tokenize.cpp

```diff
@@ -1656,7 +1656,20 @@ static bool parse_word(TokenContext &ctx, Chunk &pc, bool skipcheck)
       {
          // Turn it into a keyword now
          // Issue #1460 will return "COMMENT_CPP"
+#ifdef DEBUG
```

Comment:
What is the purpose of this debug code? It does exactly the same thing as the `else` branch and does not print any debug info. Am I missing something?

Reply:
It is a better way for me to debug the statements. YES it is the same code.

Reply:
Yes, I figured it was for debugging purposes. But once we have prepared a fix for the problem, I think it does not make much sense to include the debug code in the final code that will be committed, since it does nothing different